packages/styleUtils/layout/flexbox/stories/flex.stories.tsx

```diff
@@ -29,12 +29,22 @@ storiesOf("Style utilities/Layout/Flex", module)
   .addDecorator(withReadme([readme]))
   .addDecorator(withKnobs)
   .add("default", () => (
-    <Flex>
-      <FlexItem>1</FlexItem>
-      <FlexItem>2</FlexItem>
-      <FlexItem>3</FlexItem>
-      <FlexItem>4</FlexItem>
-    </Flex>
+    <SampleContainer>
```

Comment:
This change is unrelated to the primary goal of this PR. It was just bugging me that this story looked different than the others :)